arangod/Agency/State.cpp

```diff
@@ -1553,10 +1553,13 @@ uint64_t State::toVelocyPack(index_t lastIndex, VPackBuilder& builder) const {
   auto bindVars = std::make_shared<VPackBuilder>();
   { VPackObjectBuilder b(bindVars.get()); }
 
-  std::string const querystr 
-    = "FOR l IN log FILTER l._key <= 'buf" + stringify(lastIndex) +
-      "' SORT l._key RETURN {'_key': l._key, 'timestamp': l.timestamp,"
-                            "'clientId': l.clientId, 'request': l.request}";
+  static std::string const logstr(
+    "FOR l IN log FILTER l._key <= '%s' SORT l._key RETURN {'_key': l._key, "
+    "'timestamp': l.timestamp, 'clientId': l.clientId, 'request': l.request}");
```

Comment:
please just do `RETURN l`. I would like to have the term and other fields as well.